integrated_channel_plugin_sapsf/models.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from model_utils.models import TimeStampedModel, TimeFramedModel
+
+
+# Create your models here.
+
+class OCNWebServicesConfiguration(TimeStampedModel):
+    """
+    The Global configuration for integrating with SuccessFactors.
+    """
+
+    completion_status_api_path = models.CharField(max_length=255)
+    course_api_path = models.CharField(max_length=255)
+    oauth_api_path = models.CharField(max_length=255)
+    provider_id = models.CharField(max_length=100, default='EDX')
+
+    class Meta:
+        app_label = 'integrated_channel_plugin_sapsf'
+
+
+class OCNWebServicesEnterpriseCustomerConfiguration(TimeStampedModel):
+    """
+    The Enterprise specific configuration we need for integrating with SuccessFactors.
+    """
+
+    enterprise_customer_uuid = models.UUIDField(unique=True)
+    sapsf_base_url = models.CharField(max_length=255)
+    key = models.TextField(blank=True, verbose_name="Client ID")
+    secret = models.TextField(blank=True, verbose_name="Client Secret")
+
+    class Meta:
+        app_label = 'integrated_channel_plugin_sapsf'
+
+
+class CompletionStatusEventAudit(models.Model):
+    """
+    The payload we sent to SuccessFactors at a given point in time for an enterprise course enrollment.
+    """
+
+    enterprise_course_enrollment_id = models.PositiveIntegerField(blank=False, null=False)
+    sapsf_user_id = models.CharField(max_length=255, blank=False, null=False)
+    course_id = models.CharField(max_length=255, blank=False, null=False)
```

Comment:
Yes, the possibility of splitting off is one reason. Another is that conceptually, this is what SuccessFactor's course identifier is for our course. But we are the ones who tell them what the course identifiers are so they will likely be the same unless there is some reason we need to modify them for sending to SuccessFactors.